template-virtual-organization.yaml

```diff
@@ -85,3 +85,16 @@ SupportURL: http://www.usatlas.bnl.gov/index.shtml
 #   # - <URL>
 #   # - ...
 
+### DataFederations (optional) is information about data federations that the VO is part of
+### This template is for the StashCache data federation; for instructions,
+### see the documentation at https://opensciencegrid.org/docs/data/stashcache/vo-data/
+# DataFederations:
+#   StashCache:
+#     Namespaces:
+#       /<VONAME>/PUBLIC:
+#         - ANY
+#     AllowedCaches:
+#       - ANY
```

Comment:
Do we want this to be the default suggestion? I'm inclined to say that we should just have:

```
- <CACHE_RESOURCE>
- ...
```

Reply:
Done